.github/workflows/github.yml

```diff
@@ -13,6 +13,7 @@ jobs:
   build:
     name: Build (${{ matrix.platform }})
     strategy:
+      fail-fast: false
```

Comment:
don't forget to remove

Reply:
I have added it to not cancel all jobs when single build fails.

Reply:
it was like that intentionally

Reply:
seems that because of this change Action run is shown as green, although there's a failing mingw job. In PR the failure is visible though.

Reply:
Thats because another change to keep installer job running. In my branch it works just fine. Will investigate it in new branch and PR to see differencies.